stable/metricbeat/templates/podsecuritypolicy.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.pspEnabled }}
-apiVersion: extensions/v1beta1
+apiVersion: policy/v1beta1
```

Comment:
There are people using Kubernetes versions that don't support this. Could you please make this configurable. There are examples in the charts repo.